hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -5107,6 +5140,37 @@ public void validateReplicationConfig(ReplicationConfig replicationConfig)
     }
   }
 
+  private void initOrUpdateDirDeletingService(OzoneConfiguration conf) {
```

Comment:
the owner of dirDeletingService is KeyManagerImpl, so OM should not have callback and handling here.